routes/tx.js

```diff
@@ -164,8 +164,10 @@ router.get('/:tx', function(req, res, next) {
       tx.gasUsed = receipt.gasUsed;
       if (!tx.to) {
         tx.contractAddress = receipt.contractAddress;
-      }
-    }
+      }
+    } else {
+      tx.gasUsed = 0;
```

Comment:
I guess this makes sense because "// pending transactions from geth have null receipts", supposedly we added geth support here and missed this case.